src/index.html

```diff
@@ -1,37 +1,58 @@
-    </svg>
-  </aside>
-</body>
-</html>
+   <head>
+      <meta charset="utf-8" />
+      <title>Tema Formular</title>
+      <link rel="stylesheet" href="./style.css" />
+   </head>
+   <body>
+      <main>
+         <h2>Implementare</h2>
+         <form class="form contact-form">
+            <div>
+               <label class="form-label">Name </label>
```

Comment:
Elementul `<label>` trebuie conectat la un câmp. Fie pui input-urile în acest element, fie folosești atributul `for`